### PR TITLE
a few minor fixes

### PR DESCRIPTION
**Pull request by rlogiacco**

### CODE REVIEW

File: io.sloeber.core/src/io/sloeber/core/tools/messages.properties [hunk 1]
```diff
@@ -13,8 +13,8 @@ Helpers_IO_exception=\ I/O exception.
 Helpers_menu=menu.
 Helpers_No_boards_txt_found=No boards.txt files found in the arduino hardware folders. I looked in:\n
 Helpers_The_project=The project
-Helpers_The_folder=The folder
-Helpers_tool_reference_missing=failed to find ${TOOL} reference in ${FILE} for ${BOARD}:
+Helpers_The_folder=The folder '
+Helpers_tool_reference_missing=Ffiled to find ${TOOL} reference in ${FILE} for ${BOARD}:
```
Comment (jantje) [inline io.sloeber.core/src/io/sloeber/core/tools/messages.properties]: are you sure this is better?

**Comment by jantje** [comment]
looks like you fixed #641

**Comment by jantje** [comment]
it looks like the tests are being run and  fail :-)
Huge progress for a few minor fixes.

File: io.sloeber.core/src/io/sloeber/core/tools/messages.properties (hunk 1 above)
Comment (rlogiacco) [reply io.sloeber.core/src/io/sloeber/core/tools/messages.properties]: That is what I believe... 👅

**Pull request merged**